contrib/pg_tde/documentation/CONTRIBUTING.md

```diff
@@ -1,10 +1,90 @@
+# Contributing guide
+
+Welcome to `pg_tde` - the Transparent Data Encryption extension for PostgreSQL!
+
+We're glad that you would like to become a community member and contribute to this project.
+
+You can contribute in one of the following ways:
+
+1. Reach us on our [Forums](https://forums.percona.com/c/postgresql/25).
+2. Submit a bug report or a feature request
+3. Submit a pull request (PR) with the code patch
+4. Contribute to documentation
+
+## Prerequisites
+
+Before submitting code contributions, we ask you to complete the following prerequisites.
+
+### 1. Sign the CLA
+
+Before you can contribute, we kindly ask you to sign our [Contributor License Agreement](https://cla-assistant.percona.com/&lt;linktoCLA>) (CLA). You can do this in on click using your GitHub account.
+
+**Note**:  You can sign it later, when submitting your first pull request. The CLA assistant validates the PR and asks you to sign the CLA to proceed.
+
+### 2. Code of Conduct
+
+Please make sure to read and agree to our [Code of Conduct](https://github.com/percona/community/blob/main/content/contribute/coc.md).
+
+## Submitting a pull request
+
+All bug reports, enhancements and feature requests are tracked in [Jira](https://perconadev.atlassian.net/jira/software/c/projects/PG/issues). Though not mandatory, we encourage you to first check for a bug report among the issues and in the PR list: perhaps the bug has already been addressed.
+
+For feature requests and enhancements, we do ask you to create a Jira issue, describe your idea and discuss the design with us. This way we align your ideas with our vision for the product development.
+
+If the bug hasn’t been reported / addressed, or we’ve agreed on the enhancement implementation with you, do the following:
+
+1. [Fork](https://docs.github.com/en/github/getting-started-with-github/fork-a-repo) this repository
+2. Clone this repository on your machine. 
+3. Create a separate branch for your changes. If you work on a Jira issue, please follow this pattern for a branch name: `<PG-123>-name`. This makes it easier to track your contribution.
+4. Make your changes. Please follow the following guidelines to improve code readability:
+
+    - [PostgreSQL coding conventions](https://www.postgresql.org/docs/current/source.html)
+    - [C style and Coding rules](https://github.com/MaJerle/c-code-style) 
```

Comment:
https://github.com/MaJerle/c-code-style contradicts PG's conventions at times. For example with "_Opening curly bracket is always at the same line as keyword (for, while, do, switch, if, ...)_" as in PG they are always at the new line.